org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/tests/test.js

```diff
@@ -0,0 +1,107 @@
+var test = require('ava');
+var q = require('q');
+var OpenT2T = require('opent2t').OpenT2T;
+var config = require('./testConfig');
+
+console.log('Device Under Test -  Name: ' + config.Device.name + '  Props: ' + JSON.stringify(config.Device.props));
+var translatorPath = require('path').join(__dirname, '..');
+
+///
+/// Run a series of tests to validate the translator
+///
+
+// Get AmbientTemperature
+test.serial('AmbientTemperature', t => {

```

Comment:
Thanks, that makes sense.

---
In reply to: [75397690](https://github.com/openT2T/translators/pull/56#discussion_r75397690) [](ancestors = 75397690)